types/spdx-correct/spdx-correct-tests.ts

```diff
@@ -0,0 +1,3 @@
+import * as spdxCorrect from 'spdx-correct';
```

Comment:
please use `import spdxCorrect = require('spdx-correct')` to import commonjs modules, which use the `export = ` syntax.

Reply:
Done